debug/index.html

```diff
@@ -39,6 +39,29 @@
         task.to = new Date(task.to.getTime() + 1000 * 60 * 60 * 24);
         roadmappy.render();
       });
+      roadmappy.on('drag:start:task:from', drag.bind(null, 'from'));
+      roadmappy.on('drag:drag:task:from', drag.bind(null, 'from'));
+      roadmappy.on('drag:end:task:from', drag.bind(null, 'from'));
+      roadmappy.on('drag:start:task:to', drag.bind(null, 'to'));
+      roadmappy.on('drag:drag:task:to', drag.bind(null, 'to'));
+      roadmappy.on('drag:end:task:to', drag.bind(null, 'to'));
+      function drag(key, task, pos) {
+        const xScale = roadmappy.canvas.xScale;
+        task[key] = xScale.invert(pos.x);

```

Comment:
I would not like to support `time` format.
So I tried to fix to
```
const target = xScale.invert(pos.x);
task[key] = new Date(target.getFullYear() + '-' + (target.getMonth() + 1) + '-' + target.getDate());
```
but it sometimes has a bug that will be added 9 hour.